rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -12662,9 +12693,8 @@ void opcode_0415(const ScriptArguments& args, const ScriptVehicle vehicle, const
 	@arg arg1 
 */
 void opcode_0417(const ScriptArguments& args, const ScriptInt arg1) {
-	RW_UNIMPLEMENTED_OPCODE(0x0417);
-	RW_UNUSED(arg1);
-	RW_UNUSED(args);
+	auto offset = args.getVM()->getFile()->getMissionOffsets()[arg1];
```

Comment:
Beyond the scope: We probably want some bound checking here or the script could do malicious things